src/fusiontrees/manipulations.jl

```diff
@@ -1020,11 +1013,9 @@ function braid(f₁::FusionTree{I}, f₂::FusionTree{I},
         end
     else
         if usebraidcache_nonabelian[]
-            u = one(I)
             T = BraidingStyle(I) isa NoBraiding ?
-                typeof(Fsymbol(u, u, u, u, u, u)[1, 1, 1, 1]) :
-                typeof(sqrtdim(u) * Fsymbol(u, u, u, u, u, u)[1, 1, 1, 1] *
-                       Rsymbol(u, u, u)[1, 1])
+                Feltype(I) :
+                typeof(one(Feltype(I)) * one(Reltype(I)) * sqrtdim(one(I))) # do we need sqrtdim here ?
```

Comment:
We probably don't need `sqrtdim`. For nonabelian sectors, the `Feltype` should already be a floating point type probably.